c++/regenerate-bootstraps.sh

```diff
@@ -7,4 +7,5 @@ export PATH=$PWD/bin:$PWD:$PATH
 capnp compile -Isrc --no-standard-import --src-prefix=src -oc++:src \
     src/capnp/c++.capnp src/capnp/schema.capnp \
     src/capnp/compiler/lexer.capnp src/capnp/compiler/grammar.capnp \
-    src/capnp/rpc.capnp src/capnp/rpc-twoparty.capnp src/capnp/persistent.capnp
+    src/capnp/rpc.capnp src/capnp/rpc-twoparty.capnp src/capnp/persistent.capnp \
```

Comment:
Feel free to stick with this for now and we can file a bug to try to improve it later. Getting it to work as part of the build might turn out surprisingly complicated, at least with automake.